tests/run-ci-benchmarks.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+JSON_FILE="benchmark-data.json"
+
+cat << EOF >> ${JSON_FILE}
+[
+EOF
+
+DIRNAME=$(dirname "$0")
+
+SUCC=$(find "${DIRNAME}"/cn -name '*.c' | grep -v '\.error\.c')
+
+COUNT=0
+for TEST in ${SUCC}; do
+  let COUNT=${COUNT}+1
+done
```

Comment:
```bash
LEN=${#SUCC[@]}
```

Reply:
I spent a while attempting to get this to work without success. I don't think it's a proper array although I don't know bash that well. 

I'm inclined to leave the script as is for now and switch to a proper benchmarking system later.

Reply:
That's fine (the issue is that there needs to be an extra pair of parens on the rhs of the defining TESTS).